subprojects/internal-instrumentation-processor/src/main/java/org/gradle/internal/instrumentation/model/CallableOwnerInfo.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.instrumentation.model;
+
+import org.objectweb.asm.Type;
+
+public class CallableOwnerInfo {
```

Comment:
I wonder if we also need something like that for return types in the signature, not just for the owner.
A subtype may specify a narrower return type, and we might need to handle that in method descriptor matching...

Reply:
Wrapped also return type, but for no extra functionality for now.